include/flatbuffers/flatbuffers.h

```diff
@@ -1557,6 +1557,7 @@ class Table {
 // C++ data structures).
 // Contains no functionality, purely documentative.
 struct NativeTable {
+  virtual ~NativeTable() {}
```

Comment:
we would like to keep the base objects without overhead of a virtual table, hence why the unions work the way they work.